seedlink_plotter/config_drum.txt

```diff
@@ -0,0 +1,133 @@
+--- Configuration File (Drum Plot) ---
+
+
+----------------------------------------------------------------------------
+line_plot = False
+	# Regular real time plot for single station.
+----------------------------------------------------------------------------
+
+############################################################################
+
+# All about the data:
+----------------------------------------------------------------------------
+----------------------------------------------------------------------------
+seedlink_streams=GE_STU:BHZ   
+	# The seedlink stream selector string.
+		; Format:stream1[:selectors1],stream2[:selectors2],...
+                ; "stream" 'in "NETWORK"_"STATION" format 
+		; "selector" a space separated "list of "LOCATION""CHANNEL",
+----------------------------------------------------------------------------
+seedlink_server=geofon.gfz-potsdam.de:18000
+	# The seedlink server to connect to with port
+----------------------------------------------------------------------------
+update_time = 10  
+	# 'time in seconds between each graphic update.'
+        	; The following suffixes can be used as well: 
+			; "s" for seconds
+                        ; "m" for minutes
+			; "h" for hours
+			; "d" for days.'
```

Comment:
Please use `#` consistently for comments and get rid of these strange indentations, I think all comment lines should simply start at start of line.

Reply:
Also, usually comments come *before* the thing they comment on.